packages/manager/apps/pci-kubernetes/src/pages/detail/Detail.page.tsx

```diff
@@ -46,32 +48,31 @@ export default function DetailPage() {
       to: useResolvedPath('restrictions').pathname,
     },
     {
-      name: 'kube_logs',
+      name: 'kube_logs_tab_title',
       title: (
-        <>
+        <span>
           {tDetail('kube_logs_tab_title')}
           <OsdsChip
             className="ml-4"
             color={ODS_THEME_COLOR_INTENT.success}
             size={ODS_CHIP_SIZE.sm}
             inline
           >
-            <OsdsText
-              color={ODS_THEME_COLOR_INTENT.text}
-              level={ODS_TEXT_LEVEL.body}
-              size={ODS_TEXT_SIZE._200}
-            >
-              {tDetail('kube_logs_tab_title_beta_status')}
-            </OsdsText>
+            {tDetail('kube_logs_tab_title_beta_status')}
           </OsdsChip>
-        </>
+        </span>
       ),
       to: useResolvedPath('logs').pathname,
     },
   ];
 
   const { data: kubeDetail } = useKubeDetail(projectId, kubeId);
 
+  useEffect(() => {
+    const activeTab = tabs.find((tab) => tab.to === location.pathname);
```

Comment:
```suggestion
    const activeTab = tabs.find((tab) => location.pathname.startsWith(tab.to));
```

for the log sub section to be displayed in the correct tab